### ISSUE TITLE
feat:  more thorough reset

### ISSUE THREAD

**Issue by 9ary** [eza-community/eza#731]
I prefer to keep things minimally colored, and I would rather not have to update my configuration to remove new colors.

Currently I've got a pretty exhaustive EZA_COLORS configured, but I would rather just have a single option to remove all colors so I can then add my own back. There is already `reset` to clear the built-in file globs, another keyword for the rest would probably work?

**Comment by zbits** [comment]
Agreed.  I just installed eza and am looking for a way to clear all color formatting, not reset it back to compiled in defaults.  An option like EZA_COLORS="clear" or "delete" or "*=0" etc.

**Comment by 9ary** [comment]
I feel like * as a special code that sets all colors sounds pretty good. There's already something similar with `sn` and `sb`.
